app/screens/settings/notifications/notifications.test.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {renderWithEverything} from '@test/intl-test-helper';
+import TestHelper from '@test/test_helper';
+
+import Notifications from './notifications';
+
+import type UserModel from '@database/models/server/user';
+import type Database from '@nozbe/watermelondb/Database';
+
+jest.mock('react-native-notifications', () => ({
+    Notifications: {
+        isRegisteredForRemoteNotifications: jest.fn(),
+    },
+}));
+
+function getBaseProps() {
```

Comment:
Type the function (`ComponentProps<typeof Notification>` ) so it is safer, and we can remove the `as const` you have in line 22.